.github/workflows/python-package.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install git+https://github.com/csinva/imodels
```

Comment:
I think this will always run tests against master instead of the current branch

Reply:
good catch, sorry about that!